scripts/swap_db.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# backup database
+docker run -it --rm --network sys_default postgres:9.4 pg_dump -h db -U postgres testifi_production > dump.sql
+
+# kill app
+make stop
```

Comment:
We also need to manually `docker rm` the db now that we took it out of `make stop`